src/FieldWarning/Assets/Model/Armory/Configs.cs

```diff
@@ -16,6 +16,27 @@
 
 namespace PFW.Model.Armory
 {
+    // TODO make this not be a copy of deck..
+    [Serializable]
+    public class ArmoryConfig
+    {
+        public List<string> LOG;
```

Comment:
we can prob remove all these and just have it as part of Unit. It is really confusing to have this and the CategoryId within Unit.